### ISSUE TITLE
Problem about the training.

### ISSUE THREAD

**Issue by jianghaojun** [LeapLabTHU/Pseudo-Q#7]
Recently, several researchers asked me questions about training. The phenomenon is that the training loss did not decrease or the validation acc was very low.

*The reason might be that they adopted a smaller batch size, e.g., 96, but did not change the learning rate.*

First of all, I strongly recommend using the same batch size to reproduce our work. Secondly, if you use a smaller batch size, please try to use a smaller learning rate.

If you have any new problems with training, please post your questions inside this issue or open a new one. It would be better to provide as much information as you can, which can help me understand your question quicker.
